examples/utils/import-testing/.mocharc.cjs

```diff
@@ -8,4 +8,5 @@
 const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
 
 const config = getFluidTestMochaConfig(__dirname);
+config.timeout = 20000;
```

Comment:
Locally these took around 2.3 seconds for me, but CI showed them at 8-12 seconds. 20 second timeout should be enough for CI